package.json

```diff
@@ -132,8 +132,13 @@
     "@esm-bundle/chai": "^4.3.4",
     "@release-it/conventional-changelog": "^7.0.2",
     "@types/chai": "^4.3.20",
+    "@types/concat-stream": "^2.0.3",
+    "@types/debug": "^4.1.12",
+    "@types/minimist": "^1.2.5",
```

Comment:
I think we could drop minimis in favor of nodejs [parseArgs](https://nodejs.org/api/util.html#utilparseargsconfig) but maybe we can do that in another PR.